libs/fixtures/data/electionMinimalExhaustiveSample/cvrFiles/cdf_cast_vote_record_report.json

```diff
@@ -0,0 +1,400 @@
+        },
+        {
+          "@type": "CVR.Candidate",
+          "@id": "triggerfish",
+          "Name": "Triggerfish",
+          "PartyId": "1"
+        }
+      ],
+      "Contest": [
+        {
+          "@type": "CVR.CandidateContest",
+          "@id": "best-animal-fish",
+          "Name": "Best Animal",
+          "NumberElected": 1,
+          "VotesAllowed": 1,
+          "ContestSelection": [
+            {
+              "@type": "CVR.CandidateSelection",
+              "@id": "seahorse",
+              "CandidateIds": ["seahorse"]
+            },
+            {
+              "@type": "CVR.CandidateSelection",
+              "@id": "salmon",
+              "CandidateIds": ["salmon"]
+            }
+          ],
+          "PrimaryPartyId": "1"
+        },
+        {
+          "@type": "CVR.CandidateContest",
+          "@id": "aquarium-council-fish",
+          "Name": "Zoo Council",
+          "NumberElected": 2,
+          "VotesAllowed": 2,
+          "ContestSelection": [
+            {
+              "@type": "CVR.CandidateSelection",
+              "@id": "manta-ray",
+              "CandidateIds": ["manta-ray"]
+            },
+            {
+              "@type": "CVR.CandidateSelection",
+              "@id": "pufferfish",
+              "CandidateIds": ["pufferfish"]
+            },
+            {
+              "@type": "CVR.CandidateSelection",
+              "@id": "rockfish",
+              "CandidateIds": ["rockfish"]
+            },
+            {
+              "@type": "CVR.CandidateSelection",
+              "@id": "triggerfish",
+              "CandidateIds": ["triggerfish"]
+            },
+            {
+              "@type": "CVR.CandidateSelection",
+              "@id": "write-in-0",
+              "isWriteIn": true
+            },
+            {
+              "@type": "CVR.CandidateSelection",
+              "@id": "write-in-1",
+              "isWriteIn": true
+            }
+          ],
+          "PrimaryPartyId": "1"
+        },
+        {
+          "@type": "CVR.BallotMeasureContest",
+          "@id": "fishing",
+          "Name": "Ballot Measure 3",
+          "ContestSelection": [
+            {
+              "@type": "CVR.BallotMeasureSelection",
+              "@id": "yes",
+              "Selection": "yes"
+            },
+            {
+              "@type": "CVR.BallotMeasureSelection",
+              "@id": "no",
+              "Selection": "no"
+            }
+          ],
+          "PrimaryPartyId": "1"
+        }
+      ],
+      "ElectionScopeId": ["sample-county"]
+    }
+  ],
+  "GpUnit": [
+    {
+      "@type": "CVR.GpUnit",
+      "@id": "sample-county",
+      "Name": "Sample County",
+      "Type": "other"
+    },
+    {
+      "@type": "CVR.GpUnit",
+      "@id": "precinct-1",
+      "Name": "Precinct 1",
+      "Type": "precinct"
+    }
+  ],
+  "Batch": [
+    {
+      "@type": "VX.Batch",
+      "@id": "batch-2",
+      "BatchLabel": "Batch 2"
+    }
+  ],
```

Comment:
Should be `vxBatch`, this isn't a part of the CDF spec but I think we might as well take the opportunity the format presents to normalize the batch label data instead of repeating it.